website/server/controllers/api-v3/tasks.js

```diff
@@ -9,71 +9,17 @@ import {
   NotAuthorized,
   BadRequest,
 } from '../../libs/api-v3/errors';
+import {
+  _createTasks,
```

Comment:
If these are being exported to a lib, they should not be designated as private methods.